### DISCUSSION TITLE
Auto Title Not Working at User Level

### DISCUSSION THREAD

**Discussion by teknotut** [open-webui/open-webui#6037] [General]
I am using the web UI version V0.3.32. The auto title feature is not functioning correctly at the user level, while it works perfectly at the admin level.

<img width="349" alt="Screenshot 2024-10-09 at 06 44 10" src="https://github.com/user-attachments/assets/bfd00f18-6e97-413b-b05e-5d3931e22894">

I am not sure if other users are experiencing the same issue or if it's just me.

Please investigate this issue at your earliest convenience.

Thank you!

**Comment by tjbck** [comment]
Cannot reproduce, keep us updated!

**Reply by PretzelVector** [reply]
Hello,

I am able to reproduce this on v0.3.32. The bug matches the reproduction steps enVias provided below.

Thanks again,
~ PV

**Reply by thiswillbeyourgithub** [reply]
It would be great if we could tell openwebui to automatically retry naming chats that were not named because of llm failure. Over the months I had sometimes issues with ollama and end up with many unamed chats.

**Comment by enVias** [comment]
Reproduced when the model whitelist is in use and the title generation model has been manually selected, but the title generation model isn't whitelisted for users.

**Reply by silentoplayz** [reply]
I can confirm this issue, but when the title generation model is set to `Private` for its permission level, meaning, user accounts don't have access to use that model.
